contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,244 @@
+facilities for signaling approval or rejection of a proposed change to Kubernetes
+since anyone can open a GitHub issue at any time. Additionally managing a proposed
+change across multiple releases is somewhat cumbersome as labels and milestones
+need to be updated for every release that a change spans which leads to an ever
+increasing number of issues open against `kubernetes/features` which itself has
+become a management problem.
+
+We have chosen to use Git as our VCS with GitHub currently used to host our
+repositories. While GitHub provides a rich feature set as an open source project,
+managing issues over time and searching for text within an issue can be
+challenging. The flat hierarchy of issues can also make navigation and
+categorization tricky. While not all community members might not be comfortable
+using Git directly, I believe it is imperative that as a community we work to
+educate people on a standard set of tools so they can take their experience to
+other projects they may decide to work on in the future.
+
+It is my hope that by increasing the requirements for proposing a change to
+Kubernetes we can prevent important work like generating release notes from
+falling out of the release cycle and in general make working with and speaking
+about Kubernetes an easier and more pleasant experience.
+
+[road to Go 2]: https://blog.golang.org/toward-go2
+
+## Detailed Design
+
+### RFC Template
+
+### Git and GitHub Implementation
+
+Practically an RFC would be implemented as a pull request to a central repository
+with the following example structure
+
+```
+├── 0000-rfc-template.md
+├── CODEOWNERS
+├── index.md
+├── sig-architecture
+├── sig-network
+│   └── kube-dns
+├── sig-node
+│   └── kubelet
+├── sig-release
+├── sig-storage
+├── unsorted-to-be-used-by-newcomers-only
+└── wg-resource-management
+```
+
+where each SIG or working group is given a top level directory with components
+maintained by the SIG listed in sub directories. For newcomers to the community
+an `unsorted-to-be-used-by-newcomers-only` directory may be used before an RFC
+can be properly routed to a SIG although hopefully if discussion for a potential
+RFC begins on the mailing lists proper routing information will be provided to
+the RFC author. Additionally a top level index of RFCs may be helpful for people
+looking for a complete list of RFCs. There should be basic CI to ensure that a
+`index.md` remains up to date.
+
+Ideally no work would begin on an RFC before it has been approved by the
+responsible SIG or working group. In order to help combat the explosion of GitHub
+issues against `kubernetes/kubernetes` it is further proposed for each chartered
+SIG to create a top level repository within the `kubernetes` GitHub organization
+where implementation issues would be filed and linked to in the RFC.
+
+### RFC Editor Role
+
+Taking a cue from the [Python PEP process][], I believe that a group of RFC editors
```

Comment:
Change the tone of the document to remove the first person.  After something like this is accepted it is owned by everyone and opinions should be minimized.  Beyond that, you don't have your name on the doc anywhere so it is unclear who "I" is.